docker/pulsar/pom.xml

```diff
@@ -81,8 +104,7 @@
                         </args>
                         <contextDir>${project.basedir}</contextDir>
                         <tags>
-                          <tag>latest</tag>
-                          <tag>${project.version}</tag>
```

Comment:
I don't think that we need to remove these tags. It could be useful to keep them. 

For consistency, the new tag should also be added to all other images that are built.
for example 
`docker/pulsar-all/pom.xml`
`tests/docker-images/latest-version-image/pom.xml`
`tests/docker-images/java-test-image/pom.xml`.

`tests/docker-images/latest-version-image/Dockerfile` should reference the parent image in a similar  as `docker/pulsar-all/Dockerfile`

Reply:
That makes sense to me. I added the `latest` tag back.